plugins/unifying/unifying.quirk

```diff
@@ -36,3 +36,7 @@ Flags = is-bootloader,is-texas
 FirmwareSizeMin = 0x4000
 CounterpartGuid = HIDRAW\VEN_046D&DEV_C52B
 InstallDuration = 18
+
+# Possible HID++ v2.0 peripheral device
+[DeviceInstanceId=HIDRAW\VEN_046D]
+Plugin = unifying
```

Comment:
This will grab every Logitech device that exports a hidraw node which is not good. We need a way to check for a range here, particularly we need to check if the device is in the `4***` range.

Reply:
Well, we're whitelisting on stuff "on the lvfs" so I think that's a pretty good filter without a whitelist.